starters/oasp4j-web-starter/src/main/java/io/oasp/starter/web/config/WebAutoConfiguration.java

```diff
@@ -0,0 +1,12 @@
+package io.oasp.starter.web.config;
+
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * @author ABCHANDE
```

Comment:
Update your IDE seetings. We had manually stripped all @author comments from the code-base. Please do not re-introduce them. Instead better write JavaDoc for the rationale of this class. If all these AutoConfigs do nothing - why do we have them in place? I would only add them if they actually do something.